spec/factories/users.rb

```diff
@@ -0,0 +1,12 @@
+FactoryGirl.define do
+  sequence(:email) { |n| "bg#{n}@pipedpiper.net" }
+  factory :user do
+    first_name "Bertram"
+    last_name "Guilfoyle"
+    email
+    profile_picture "photo.png"
+    role "member"
+    password "SatanCanada"
```

Comment:
Password Confirmation is required. For creating a new user it has to have password confirmation even for the Factory.